tbconnect/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
We can probably remove this file, and instead rely on the separate `test_views.py` etc